### DISCUSSION TITLE
Next release?

### DISCUSSION THREAD

**Discussion by jaaneh** [nextauthjs/next-auth#940] [Help] [answered]
Hello! Thanks for the great product.

I was curious when the next release will be? I'm trying to use the Bungie provider merged in #589.

Trying to add this as a custom provider and having some issues as well, so just curious when you'll be pushing a new release.

**Comment by balazsorban44** [comment, marked as answer]
Hi there, we are aiming for one `canary` (unstable) release in December! Stable will have to wait a little bit, as we want to improve our tests first. 

In the meantime, you should be able to try things out, by using the `canary` branch as the source and build it your own.

Something like this. https://medium.com/@jonchurch/use-github-branch-as-dependency-in-package-json-5eb609c81f1a

**Reply by jaaneh** [reply]
Much appreciated! Cheers
